CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `argilla.tasks.database.migrate` new task to execute database migrations ([#2564]).
 - `release.Dockerfile` and `quickstart.Dockerfile` now creates a default `argilladata` volume to persist data ([#2564]).
 - Add user settings page. Closes [#2496](https://github.com/argilla-io/argilla/issues/2496)
+- Hide the user token in user settings. See [#2652](https://github.com/argilla-io/argilla/pull/2652)
```

Comment:
Keep in mind the purpose of this file: Show relevant changes between versions. The second line is related to an addition that is already included in the first line. 

Also, changes on this PR do not add anything concerning the previous release. So, I would prefer if you just remove the line.